logger.go

```diff
@@ -39,8 +39,12 @@ type Logger struct {
 	entryPool sync.Pool
 	// Function to exit the application, defaults to `os.Exit()`
 	ExitFunc exitFunc
+	// ErrorCallback is called when any error occurs in Logrus
```

Comment:
If I'm mistaken ErrorCallback is only called when a formatter or hook returns an error and there are other places where errors could happen no?

Reply:
It would be good to also tell developers using logrus what the default is (it does nothing, but I can't assume that).